addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1148,6 +1149,349 @@ registry.anchorSlide = publicWidget.Widget.extend({
     },
 });
 
+
+registry.countdown = publicWidget.Widget.extend({
+    selector: '.s_countdown',
+    xmlDependencies: ['/website/static/src/xml/website.s_countdown.xml'],
+    disabledInEditableMode: false,
+
+    /**
+     * @override
+     */
+    start: function () {
+        this.hereBeforeTimerEnds = false;
+        this.endAction = this.$el[0].dataset.endAction;
```

Comment:
`this.$el[0]` => `this.el`

Reply:
Done, didn't know about that one, thanks